data/wastewater_virus.csv

```diff
@@ -1,689 +0,0 @@
-<<<<<<< HEAD

```

Comment:
Should this data be kept for the version 1.0.0 and 1.1.0 archives?

Reply:
The data is archived at https://github.com/Big-Life-Lab/PHES-ODM/releases/tag/v1.0.0. (We need to add a V1.1 release but there is a GH tag for that release). 

The releases and archives can be hard to find but we can make links to them in the documentation, etc.

Reply:
Completed. We will use the releases to keep a copy of all version assets. 
Note that we'll want to add a few assets to the release archive that are outside this GitHub repo.
- The Excel spreadsheet and templates on osf.io.  https://osf.io/49z2b/
- a copy of the documentation book. https://github.com/Big-Life-Lab/PHES-ODM-doc
- links to the release versions of the toolkit, such as the validation library? https://github.com/Big-Life-Lab/PHES-ODM-Validation
- Other items?